### DISCUSSION TITLE
zh-cn/protocol/base-scheduling-schema

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#139] [Comments]
# zh-cn/protocol/base-scheduling-schema

基建排班协议 resource/custom_infrast/*.json 的使用方法及各字段说明 提示 请注意 JSON 文件是不支持注释的，文本中的注释仅用于演示，请勿直接复制使用 可视化排班生成工具 完整字段一览 举例 243 极限效率，一天三换 153 极限效率，一天三换

https://docs.maa.plus/zh-cn/protocol/base-scheduling-schema.html

**Comment by ShrLeeKNsword** [comment]
有没有大佬写份作业，急急急

**Comment by godthis** [comment]
训练室的key值是多少啊

**Reply by ABA2396** [reply]
训练室只有继续专精，没有自定义放人

**Comment by YozonChen** [comment]
求求增加训练室的自定义放人功能

**Comment by wigreal** [comment]
我这边更新队列更换后，就完全不安排人入住宿舍睡觉了，这是什么原因呢？

**Comment by remilia-BF** [comment]
1728779152850250 留一个，新版本好像更新了，留个中配243

**Comment by pistolplayer** [comment]
你好，我这边贸易站如果选择跳过房间，就无法使用无人机加速，但是如果不选择跳过房间，就会重新选择一次干员，这会让我的巫恋组重新暖机，有什么解决方法吗

**Reply by ABA2396** [reply]
用自定义基建配置
